### PR TITLE
Feature/task2

### PR DESCRIPTION
**Pull request by Gelelus**
## S3 bucket has been created and configured properly. The app has been uploaded to the bucket and is available though the Internet. Nothing else has been done.

Link to S3 bucket - https://s3.console.aws.amazon.com/s3/buckets/gelelus-rs-first/?region=us-east-1&tab=overview
Link to S3 website - http://gelelus-rs-first.s3-website-us-east-1.amazonaws.com

##  In addition to the previous work a CloudFront distribution is created and configured properly and the site is served now with CloudFront and is available through the Internet over CloudFront URL, not S3-website link (due to changes in bucket’s policy...). 

Link to CloudFront website - https://d4vjyxpw6ibcq.cloudfront.net/
Link to S3 website - http://gelelus-rs-app-first.s3-website-us-east-1.amazonaws.com/

## Serverless-finch and serverless-single-page-app plugins are added and configured. The app can be built and deployed by running npm script command.

Link to CloudFront website - https://dlvllyd9xm1fd.cloudfront.net/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,7 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+
+# Serverless directories
+.serverless
```
Comment (postc0de) [inline .gitignore:31]: no need to add it a second time, as it is already there by default (see line#13)

**Review by postc0de — COMMENTED** [review]
Well done!

**Review by postc0de — APPROVED** [review]

**Comment by pavel-leskavets** [comment]
Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?

**Comment by uniorunr** [comment]
It seems that http://gelelus-rs-app-first.s3.amazonaws.com/ is not a correct link. Are you meant http://gelelus-rs-app-first.s3-website-us-east-1.amazonaws.com/ ?
And as mentioned above, http://gelelus-rs-first.s3-website-us-east-1.amazonaws.com/ shouldn't be available. It should throw 403

**Comment by Gelelus** [comment]
> 
> 
> It seems that http://gelelus-rs-app-first.s3.amazonaws.com/ is not a correct link. Are you meant http://gelelus-rs-app-first.s3-website-us-east-1.amazonaws.com/ ?
> And as mentioned above, http://gelelus-rs-first.s3-website-us-east-1.amazonaws.com/ shouldn't be available. It should throw 403

сорян не ту ссылку вставил

**Comment by Gelelus** [comment]
> 
> 
> Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?

опять таки не та ссылка

**Comment by pavel-leskavets** [comment]
> > Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?
> 
> опять таки не та ссылка

Но ссылка с ручного S3 все равно доступна, так быть не должно

**Comment by Gelelus** [comment]
> 
> 
> > > Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?
> > 
> > 
> > опять таки не та ссылка
> 
> Но ссылка с ручного S3 все равно доступна, так быть не должно



> 
> 
> > > Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?
> > 
> > 
> > опять таки не та ссылка
> 
> Но ссылка с ручного S3 все равно доступна, так быть не должно

чел у меня на s3 3 бакета 
1 бакет для :
    3 - S3 bucket has been created and configured properly. The app has been uploaded to the bucket and is available though the Internet. Nothing else has been done. (**Link to S3 bucket/website is provided**. There is no Pull Request in the YOUR OWN frontend repository.) - как раз таки тут нужна ссылка на s3 доступный бакет и доступную веб версии
2 бакет для:    
 4 - In addition to the previous work a CloudFront distribution is created and configured properly and the site is served now with CloudFront and is available through the Internet over CloudFront URL, not S3-website link (due to changes in bucket’s policy...). (Link to CloudFront website is provided. S3-website shows 403 Access Denied error. There is no Pull Request in the YOUR OWN frontend repository.) - по этому тоже две ссылки ручной s3 (недоступный 403 возвращает) и cloudfront 
3 бакет для: 
    5 - Serverless-finch and serverless-single-page-app plugins are added and configured. The app can be built and deployed by running npm script command. (Link to CloudFront website is provided. PR with all changes is submitted in the YOUR OWN frontend repository and its link is provided for review.) - по этому ссылка автоматический деплой cloudfront 

мде

**Comment by pavel-leskavets** [comment]
> > > > Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?
> > > 
> > > 
> > > опять таки не та ссылка
> > 
> > 
> > Но ссылка с ручного S3 все равно доступна, так быть не должно
> 
> > > > Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?
> > > 
> > > 
> > > опять таки не та ссылка
> > 
> > 
> > Но ссылка с ручного S3 все равно доступна, так быть не должно
> 
> чел у меня на s3 3 бакета
> 1 бакет для :
> 3 - S3 bucket has been created and configured properly. The app has been uploaded to the bucket and is available though the Internet. Nothing else has been done. (**Link to S3 bucket/website is provided**. There is no Pull Request in the YOUR OWN frontend repository.) - как раз таки тут нужна ссылка на s3 доступный бакет и доступную веб версии
> 2 бакет для:
> 4 - In addition to the previous work a CloudFront distribution is created and configured properly and the site is served now with CloudFront and is available through the Internet over CloudFront URL, not S3-website link (due to changes in bucket’s policy...). (Link to CloudFront website is provided. S3-website shows 403 Access Denied error. There is no Pull Request in the YOUR OWN frontend repository.) - по этому тоже две ссылки ручной s3 (недоступный 403 возвращает) и cloudfront
> 3 бакет для:
> 5 - Serverless-finch and serverless-single-page-app plugins are added and configured. The app can be built and deployed by running npm script command. (Link to CloudFront website is provided. PR with all changes is submitted in the YOUR OWN frontend repository and its link is provided for review.) - по этому ссылка автоматический деплой cloudfront
> 
> мде

Старайтесь придерживаться более формального стиля общения, мы все же с вами не знакомы. Если у вас 3 бакета, где S3 линк для автоматического деплоя?

**Comment by Gelelus** [comment]
> 
> 
> > > > > Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?
> > > > 
> > > > 
> > > > опять таки не та ссылка
> > > 
> > > 
> > > Но ссылка с ручного S3 все равно доступна, так быть не должно
> > 
> > 
> > > > > Links from S3 shouldn't be available. They should throw 403 error. Where is S3 link from automatic deployment?
> > > > 
> > > > 
> > > > опять таки не та ссылка
> > > 
> > > 
> > > Но ссылка с ручного S3 все равно доступна, так быть не должно
> > 
> > 
> > чел у меня на s3 3 бакета
> > 1 бакет для :
> > 3 - S3 bucket has been created and configured properly. The app has been uploaded to the bucket and is available though the Internet. Nothing else has been done. (**Link to S3 bucket/website is provided**. There is no Pull Request in the YOUR OWN frontend repository.) - как раз таки тут нужна ссылка на s3 доступный бакет и доступную веб версии
> > 2 бакет для:
> > 4 - In addition to the previous work a CloudFront distribution is created and configured properly and the site is served now with CloudFront and is available through the Internet over CloudFront URL, not S3-website link (due to changes in bucket’s policy...). (Link to CloudFront website is provided. S3-website shows 403 Access Denied error. There is no Pull Request in the YOUR OWN frontend repository.) - по этому тоже две ссылки ручной s3 (недоступный 403 возвращает) и cloudfront
> > 3 бакет для:
> > 5 - Serverless-finch and serverless-single-page-app plugins are added and configured. The app can be built and deployed by running npm script command. (Link to CloudFront website is provided. PR with all changes is submitted in the YOUR OWN frontend repository and its link is provided for review.) - по этому ссылка автоматический деплой cloudfront
> > мде
> 
> Старайтесь придерживаться более формального стиля общения, мы все же с вами не знакомы. Если у вас 3 бакета, где S3 линк для автоматического деплоя?

По заданию она не требовалась(только ссылка на CloudFront ) . Или я ослеп? ниже задание 
5 - Serverless-finch and serverless-single-page-app plugins are added and configured. The app can be built and deployed by running npm script command. (**Link to CloudFront website is provided**. PR with all changes is submitted in the YOUR OWN frontend repository and its link is provided for review.)
http://gelelus-rs-app-second.s3-website-us-east-1.amazonaws.com (вот она , если нид пруф)

**Pull request merged**